components/tikv_kv/src/lib.rs

```diff
@@ -267,10 +268,19 @@ pub trait Engine: Send + Clone + 'static {
     type Local: LocalEngine;
 
     /// Local storage engine.
-    fn kv_engine(&self) -> Self::Local;
+    ///
+    /// In the single rocksdb version, the return value is guaranteed to be
+    /// `Some` where as in the multi-rocksdb version, the `None` will be
+    /// returned.
+    fn kv_engine(&self) -> Option<Self::Local>;
```

Comment:
How about `test_kv_engine` or `_kv_engine`, if this is only used in tests. (I'm not sure the idiomatic way to express this in Rust, cc @BusyJay)

Reply:
It will be used for v1.